src/components/ui/sheet.stories.tsx

```
@@ -0,0 +1,191 @@
+
+export const Right: Story = {
+  args: {
+    open: true,
+  },
+  render: () => (
+    <Sheet open>
+      <SheetContent side="right">
+        <SheetHeader>
+          <SheetTitle>Edit Profile</SheetTitle>
+          <SheetDescription>
+            Make changes to your profile here. Click save when you're done.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)